### DISCUSSION TITLE
Drag an drop as a way to sort customers for the property manager

### DISCUSSION THREAD

**Discussion by LucBen93** [AppStractSE/PropertyManager#85] [Ideas]
Add drag and drop to the customer view for a property manager as a way for them to manually sort customers instead of (or as an extra function) sorting by name or least amount of finished assignments.

**Comment by 4l3x91** [comment]
if on pc - very good, if on mobile - not so sure, we needs to test - decide on which dnd library to use and look for one where it's easy to drag and drop on touch screens
